rest-api-spec/src/main/resources/rest-api-spec/api/behavioral_analytics.put.json

```diff
@@ -0,0 +1,35 @@
+{
+  "behavioral_analytics.put": {
+    "documentation": {
+      "url": "http://todo.com/tbd",
+      "description": "Creates a behavioral analytics collection."
+    },
+    "stability": "experimental",
+    "visibility": "feature_flag",
+    "feature_flag": "xpack.ent-search.enabled",
+    "headers": {
+      "accept": [
+        "application/json"
+      ],
+      "content_type": [
+        "application/json"
+      ]
+    },
+    "url": {
+      "paths": [
+        {
+          "path": "/_application/analytics/{name}",
```

Comment:
nit: is it really needed? Why would we allow to create multiple collections in one call? Can we require multiple calls for this use case?

Reply:
So the endpoint shouldn't allow creating multiple collections in one call. Have I described the schema incorrectly here?